tenant/sunbird/index.html

```diff
@@ -732,10 +713,10 @@ <h4 class="sb-se-title ">Urban Domains</h4>
                     <div ngxSlickItem class="slide">
                         <div class=" d-flex domain-box">
                             <div>
-                                <img src="images/domainHousing.png" alt="Domain Housing">
+                                <img src="images/domainHousing.png" alt="Housing">
 
                             </div>
-                            <div class="domain-text">Domain Housing</div>
+                            <div class="domain-text">Housing</div>
```

Comment:
@komalm, could you please confirm whether the domain name is **Housing** or  **Urban Planning and Housing**?